mteb/tasks/Classification/heb/Hebrew_Sentiment_Analysis.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification  # type: ignore
+from mteb.abstasks.TaskMetadata import TaskMetadata  # type: ignore
```

Comment:
Did the linter add `# type: ignore` ? If no, then you can remove it imo

Reply:
Yes, the linter added it